### ISSUE TITLE
Customizing Colors for system

### ISSUE THREAD

**Issue by matthewnaples** [kiwicom/orbit-swiftui#268]
Hello, I'm just wondering if there are any plans to add custom color themes. I'd like to use my "product" color that's different than the one baked in. I'm working with a local copy of the repo and changing a couple of the hues, but I'm wondering if there's a better way to go about this? Maybe I could contribute to this if it's not on the docket. I did see a "Prepare design tokens for native platforms" item on the roadmap on your website.

**Comment by PavelHolec** [comment]
Hello,
in future, a "product" design tokens would be a way to go. Currently, the Orbit palette itself is not configurable.

A workaround for now would be to use the `.custom()` case for color-related variants on components instead of predefined colors. These `custom` enum cases are meant for such cases where a native Orbit style needs to be overridden.

**Comment by hrach** [comment]
FYI: both web and android support custom theming: https://github.com/kiwicom/orbit/blob/master/.github/theming.md

**Comment by matthewnaples** [comment]
I'll try this workaround for now. Thank you for getting back to me!

**Comment by ysfsl** [comment]
I can't wait for it to be supported like android and web style color customization for iOS. @matthewnaples

**Comment by PavelHolec** [comment]
Should be supported by #287. The Storybook contains example of color palette override

**Comment by cameroncooke** [comment]
Just wanted to follow this up as this change seems to have been reversed in https://github.com/kiwicom/orbit-swiftui/pull/715 - I feel this issue should be re-opened as I also need to overridden the default theme colors.

**Comment by cameroncooke** [comment]
One option I can think of would be to allow a custom bundle to be used so that we could then just use our own Color.xcassets

**Comment by PavelHolec** [comment]
> One option I can think of would be to allow a custom bundle to be used so that we could then just use our own `Color.xcassets`

Good idea. That would match the proposed localization bundle environment injection in #764 (either via parameter and/or via environment).
